lib/timber/log_entry.ex

```diff
@@ -241,4 +237,17 @@ defmodule Timber.LogEntry do
 
     [context, event, meta]
   end
+
+  defp vm_pid_from_metadata(metadata) do
+    vm_pid = Keyword.get(metadata, :pid, self())
+
+    case vm_pid do
+      vm_pid when is_pid(vm_pid) ->
+        vm_pid
+        |> :erlang.pid_to_list()
+        |> :erlang.iolist_to_binary()
+      vm_pid ->
+        inspect(vm_pid)
```

Comment:
I think we should use `to_string/1` instead of `inspect/1` here.

Reply:
I went with inspect as it communicates it's an atom, but that's not super important. ex:
```
to_string(:emulator)` # => "emulator"
inspect(:emulator)` # => ":emulator"
```

Will switch to `to_string/1`